deployment/docker/store/store_msg_retriever.py

```diff
@@ -55,10 +55,10 @@ def fetch_all_messages(base_url: str, initial_params: Dict, headers: Dict) -> Li
         logging.info(data)
         all_messages.extend([message['messageHash'] for message in data['messages']])
 
-        cursor = next_cursor(params)
+        cursor = next_cursor(data)
         if not cursor:
             break
-        data["cursor"] = cursor
+        params["cursor"] = cursor
```

Comment:
Quick question for my understanding. Is the idea to modify `initial_params` by reference?